msr/msre/multiphysics_core_model/steady_state/tests

```diff
@@ -0,0 +1,23 @@
+[Tests]
+  # Syntax checks
+  [coupled_msre_multiphysics_syntax]
+    type = RunApp
+    input = 'neu.i'
+    check_input = True
+    executable_pattern = 'blue_crab*'
```

Comment:
since blue_crab is so hard to keep green, it s must easier to make 2 other single-app syntax tests with Griffin and Pronghorn here. Then 2 small regression tests that are single-app as well (disabling the multiaps and transfer block using MutliApps/active='' Transfers/active='' in the cli_args)

Reply:
Done